gradle/libs.versions.toml

```diff
@@ -38,6 +38,8 @@ threetenbp = "1.5.2"
 micronaut = "3.7.4"
 micronautPicocli = "4.1.0"
 micronautJunit5 = "3.4.0"
+micronautSecurity="3.9.2"
```

Comment:
This dependency actually brings the micronaut 3.7.5 and junit 5.9.1, hence the need for public `@BeforeAll` methods, maybe we should increase these versions explicitly?

Reply:
done